src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx

```diff
@@ -93,10 +86,12 @@ function FilterShortMenuColumn({
   onClick,
 }: FilterShortMenuColumnProps) {
   const dispatch = useDispatch();
+  const { t } = useTranslation();
   const visableParams = params.length > 0;
 
   function handleMenuToggle() {
-    const tabIndex = filterName === "Season" ? 4 : 5;
+    const tabIndex =
+      filterName === "Season" ? 4 : filterName === "Brand" ? 5 : 6;
```

Comment:
Don’t use a ternary operator for the complex expression

Reply:
replaced to "switch"